.github/workflows/deploy-prod.yml

```
@@ -41,9 +46,29 @@ jobs:
             - name: Deploy Project Artifacts to Vercel
               run: vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }}
 
-            - name: Label as deployed
-              uses: buildsville/add-remove-label@v2.0.1
-              with:
-                  token: ${{ secrets.GITHUB_TOKEN }}
-                  labels: Deployed
-                  type: add
+            - name: Get PR number and apply label
+              if: github.event_name == 'push'
+              run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2016:info:1:37: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)